lib/adapters/MochaAdapter.js

```diff
@@ -29,16 +29,23 @@ export default class MochaAdapter extends EventEmitter {
   convertSuite (mochaSuite) {
     return new Suite(
       mochaSuite.title,
+      this.buildSuiteCanonicalName(mochaSuite),
```

Comment:
Now that the property is `fullName`, should update methods still using "canonical" as well.